tools/hubl/go.mod

```diff
@@ -119,3 +119,9 @@ require (
 	pgregory.net/rapid v0.5.5 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+
+// Temporal for PR
```

Comment:
Let's not forget to remove that in a follow-up of now using a pseudo version.
Otherwise hubl cannot be installed via `go install`.